notebook/services/kernelspecs/tests/test_kernelspecs_api.py

```diff
@@ -9,31 +9,30 @@
 
 pjoin = os.path.join
 
-import requests
-
 from jupyter_client.kernelspec import NATIVE_KERNEL_NAME
-from notebook.utils import url_path_join, url_escape
```

Comment:
Another case where linter may be wrong, but are probably right here. 
it's unclear wether we are just testing that `url_escape` just exists or if we are importing it. 
It may also be that importing `requests` is done in order to trigger side effects. 

I don't believe any of the two above statement are true, and this change would be fine.